training/distributed_training/tensorflow/data_parallel/mnist/infer_tensorflow.ipynb

```diff
@@ -72,62 +98,6 @@
     "predictor = model.deploy(initial_instance_count=1, instance_type='ml.m4.xlarge')"
    ]
   },
-  {
```

Comment:
Why has the endpoint test been removed?

Reply:
because so many notebooks already have the exactly the same test and I think it is better to make the content more modular. we can it by having one notebook elaborate on tensorflow model deployment and link other notebooks to it